chain/client/src/test_utils.rs

```diff
@@ -592,13 +606,11 @@ pub fn setup_mock_all_validators(
         let client_addr = ClientActor::create(|ctx| {
             let client_addr = ctx.address();
             let _account_id = account_id.clone();
-            let pm = PeerManagerMock::mock(Box::new(move |msg, _ctx| {
+            let pm = PeerManagerMock{handle:Box::new(move |msg, _ctx| {
```

Comment:
Missing whitespace. I think that's the second time we manually construct a PereManagerMock, so let's maybe add `PeerManagerMock::new` which accepts an `impl Fn` and does Boxing and type-erasure internally.

Reply:
done

Reply:
Also, that's a general pattern: you provide `impl Trait` interface at the API level, and erase to `Box<dyn Trait>` internally -- usually gives a nicer to use API, and shields the user of the details of boxing